SaneEconomyCore/src/main/java/org/appledash/saneeconomy/BukkitBootstrap.java

```diff
@@ -0,0 +1,132 @@
+package org.appledash.saneeconomy;
+
+import java.io.File;
+import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
+import java.net.URLClassLoader;
+import java.nio.file.Files;
+import java.sql.Driver;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Collections;
+import java.util.Optional;
+import java.util.logging.Level;
+import me.lucko.jarrelocator.Relocation;
+import org.appledash.saneeconomy.economy.EconomyManager;
+import org.appledash.saneeconomy.economy.logger.TransactionLogger;
+import org.appledash.saneeconomy.utils.JarUtil;
+import org.appledash.saneeconomy.utils.LogUtil;
+import org.appledash.saneeconomy.vault.VaultHook;
+import org.appledash.sanelib.SanePlugin;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.slf4j.Logger;
```

Comment:
Despite the dependency on SLF4J, this is simply a requirement of Hikari - I don't actually want to use it.

Reply:
SLF4J is actually a decent logging impl if you don't want to use services or singletons or have static references to a logging utility all over the place. If you can think of something better, then fantastic! I use slf4j because it's the de-facto standard logging library for Bukkit, Spigot, and Paper.